frontend/src/components/SlideSearchFunction.css

```diff
@@ -0,0 +1,41 @@
+.noEntrySearch,
+.enterYourSearch {
+  display: none;
+}
+
+@media screen and (max-width: 600px) {
+  .searchBarButton {
+    margin-left: 0;
+    z-index: 2;
+  }
+
+  .enterYourSearch {
+    display: block;
+    width: 80%;
+    height: calc(100vh - 100px);
+    border-radius: 5px;
+    background-color: black;
+    position: absolute;
+    top: 50px;
+    right: 10px;
+    transition-duration: 1s;
+    transform: translate(-10px, 0);
+    z-index: 1;
+    overflow: scroll;
+  }
+  .noEntrySearch {
+    display: block;
+    width: 0%;
+    background-color: black;
+    position: absolute;
+    top: 50px;
+    right: -2px;
+    transition-duration: 1s;
+    z-index: 1;
+  }
+  #searchingBarre {
```

Comment:
`#searchBar` :p
I'll be quite strict about class/id naming, 'cause it can cause a *lot* of confusion group-wise if someone uses wrong-named ones

Reply:
id change to #searchBarMobile